lib/Common/Memory/LargeHeapBlock.cpp

```diff
@@ -553,6 +553,12 @@ LargeHeapBlock::AllocFreeListEntry(DECLSPEC_GUARD_OVERFLOW size_t size, ObjectIn
 #ifdef RECYCLER_WRITE_BARRIER
     header->hasWriteBarrier = (attributes & WithBarrierBit) == WithBarrierBit;
 #endif
+
+    if ((attributes & (FinalizeBit | TrackBit)) != 0)
+    {
+        // Make sure a valid vtable is installed as once the attributes have been set this allocation may be traced by background marking
+        allocObject = (char *)new (allocObject) DummyVTableObject();
+    }
```

Comment:
HostVisited does not currently use Large blocks, right? Is the change here for consistency? If true, should line 627 also be updated?

Reply:
It does not use LargeBlocks currently (and in fact, neither does TrackBit); yes it is for consistency; yes I should update line 627. Thanks!